api/src/test/java/org/openmrs/module/fhir2/api/search/PatientSearchQueryImplTest.java

```diff
@@ -310,7 +310,7 @@ public void searchForPatients_shouldSearchForPatientsByBirthDateWithLowerBound()
 		
 		assertThat(results, notNullValue());
 		assertThat(get(results), not(empty()));
-		assertThat(get(results), hasSize(equalTo(1)));
+		assertThat(get(results), hasSize(greaterThan(1)));
```

Comment:
Not sure why this is part of this commit?

Reply:
@ibacher Earlier I had changed this so that the tests pass for openmrsPlaformVersion 2.3.0. So I reverted it back to this after creating a new sub module